backend/http_settings.go

```diff
@@ -11,12 +11,14 @@ import (
 // HTTPSettings is a convenient struct for holding decoded HTTP settings from
 // jsonData and secureJSONData.
 type HTTPSettings struct {
-	Access            string
-	URL               string
-	BasicAuthEnabled  bool
-	BasicAuthUser     string
-	BasicAuthPassword string
-	Headers           map[string]string
+	Access                string
+	URL                   string
+	BasicAuthEnabled      bool
+	BasicAuthUser         string
+	BasicAuthPassword     string
+	Headers               map[string]string
+	ForwardOAauthIdentity bool
```

Comment:
```suggestion
	ForwardOAuthIdentity bool
```

🙈

Reply:
Haven't you heard about this new thing OAauth :facepalm: